onnxruntime/core/graph/contrib_ops/bert_defs.cc

```diff
@@ -347,10 +354,32 @@ ONNX_MS_OPERATOR_SET_SCHEMA(
                " or (1, num_heads, sequence_length, total_sequence_length)",
                "T",
                OpSchema::Optional)
+        .Input(6,
+               "past_key",
+               "past state for cross key with shape (batch_size, num_heads, past_sequence_length, head_size)",
```

Comment:
remove "cross". For example, GPT-2 could also have past_key, but it is not cross attention.
What's the shape of past_key when static_kv=1?

Reply:
I think I forget to update it after enabling the self attention.
In cross-attention(static_kv=1) past_key shape is (batch, num_heads, kv_s, head_size)